packages/schema/src/schema/processing/concaveHull.json

```
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "description": "ConcaveHull",
+  "title": "IConcaveHull",
+  "required": ["inputLayer", "pctconvex"],
+  "additionalProperties": false,
+  "properties": {
+    "inputLayer": {
+      "type": "string",
+      "description": "The input layer for Concave Hull."
+    },
+    "pctconvex": {
+      "type": "number",
+      "description": "Controls the concaveness of the computed hull, vary from 0 to 1, 0 corresponds to the maximum concavity."
```

```suggestion
      "type": "number",
      "description": "Controls the concaveness of the computed hull, vary from 0 to 1, 0 corresponds to the maximum concavity."
      "default": 0.5,
      "multipleOf": 0.1,
      "minimum": 0,
      "maximum": 1
```

I don't know for the default value though, what do you think?

---

Neat! A thought for the future: let's show a loading indicator when processing is in progress! Opening an issue.

---

https://github.com/geojupyter/jupytergis/issues/765